### ISSUE TITLE
docker volumes inside envy

### ISSUE THREAD

**Issue by jgallen23** [progrium/envy#38]
I'm trying to run a container with a docker volume from inside of an envy instance, but the folders always end up empty.  Is this because of the docker inside docker?  Is there any way around it?

**Comment by progrium** [comment]
Yes it's probably going to be solved like boot2docker where the docker
instance has a mount of part of your local file system. So likely envy
dockers will have access to anything under your home and root home..

On Monday, July 6, 2015, Greg Allen notifications@github.com wrote:

> I'm trying to run a container with a docker volume from inside of an envy
> instance, but the folders always end up empty. Is this because of the
> docker inside docker? Is there any way around it?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/progrium/envy/issues/38.

## 

Jeff Lindsay
http://progrium.com

**Comment by jgallen23** [comment]
Is there any way around this right now? Which container needs to mount the root filesystem? The envy session or the dind?

**Comment by progrium** [comment]
First we need to decide what part of the session container should be shared with dind to mount with your containers. `/work` maybe?

**Comment by fgrehm** [comment]
I've been using `/workspace` on my fork

**Comment by jgallen23** [comment]
No preference on `/work` or `/workspace`
